deploy/crds/apps_v1alpha1_apimanager_crd.yaml

```diff
@@ -60,14 +65,22 @@ spec:
             system:
               properties:
                 database:
+                  description: TODO should union fields be optional?
```

Comment:
For some reason the generator now adds some fields based on some comments that were not added before.
This should not appear on the crd

Reply:
those are comments on struct and end up in crd. 

I think we should improve description instead of removing description from CRD's. The more info in the CRD's the better, right?

Reply:
I agree, but isn't there a way to keep comments that won't appear as generated? those are useful to clarify/remember some information that does not necessarily belong to the description

Reply:
According to 
https://github.com/kubernetes/kube-openapi/blob/master/pkg/generators/openapi.go#L483

Adding `---` or `+` or `TODO` should ignore, but does not work. 

We will add extra new line to remove comment